stash_engine/spec/config/app_config.yml

```diff
@@ -5,6 +5,8 @@ test:
   repository: Stash::MockRepository
   submission_bc_emails: ["alan.smithee@example.edu"]
   contact_email: ["contact1@example.edu", "contact2@example.edu"]
+  old_dryad_access_token: "this-is-not-a-real-token"
+  old_dryad_url: http://ryan-vm.datadryad.org
```

Comment:
probably not a big deal but do you want your VM as part of the example config?

Reply:
fixed